src/components/PeopleList/PeopleTile/styled.js

```diff
@@ -1,36 +1,44 @@
 import { NavLink } from "react-router-dom/cjs/react-router-dom.min";
 import { styled } from "styled-components";
 
+export const Wrapper = styled.div`
+  transition: 0.15s;
+
+  // ten efekt powoduje poruszanie się elementów w PeopleTileNavLink
+  // trzeba znaleźć rozwiązanie lub zamiennić na inny
+  &:hover { 
+    transform: scale(1.05);
+  }
+`;
+
 export const PeopleTileNavLink = styled(NavLink)`
+  width: 100%;
+  height: 100%;
   text-decoration: none;
   color: inherit;
-`;
-
-export const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 12px;
   background-color: ${({ theme }) => theme.color.white};
   padding: clamp(8px, 0.85vw , 16px);
-  transition: 0.6s;
   box-shadow: 0px 4px 12px 0px ${({ theme }) => theme.color.boxShadow};
   border-radius: 5px;
   cursor: pointer;
-  width: 100%;
-  overflow: hidden; 
-  max-height: 339px;
 
-  &:hover { 
-    transform: scale(1.05);
+  @media (max-width: ${({ theme }) => theme.breakpoint.large}) {
+    gap: clamp(8px, 2.5vw , 12px);
   }
 
   @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
     padding: 8px;
+    gap: 8px;
     min-height: 245px;
   }
 `;
 
 export const Image = styled.img`
   margin: 0;
-  margin-bottom: clamp(8px, 1vw, 12px);
-  width: 100%;
```

Comment:
I think that by removing these lines of code, the photo inside the PeopleTile does not change its size, which is why the layout breaks on smaller screens.
![cast-crew-images](https://github.com/Urbork/movies-browser/assets/129622893/8d1079cc-17c1-4a3d-bb70-1dffac445d86)